ui/tabs.js

```diff
@@ -713,7 +713,7 @@ $.widget( "ui.tabs", {
 	_getIndex: function( index ) {
 		// meta-function to give users option to provide a href string instead of a numerical index.
 		if ( typeof index === "string" ) {
-			index = this.anchors.index( this.anchors.filter( "[href$='" + index + "']" ) );
+			index = this.anchors.index( this.anchors.filter( "[href$='" + $.ui.escapeId( index ) + "']" ) );
```

Comment:
The escaping is the same and i meant to leave a comment about needing a better name.